packages/git/README.md

```diff
@@ -44,9 +44,8 @@ GIT_PRIVATE_KEY="/path/to/your/key/id_rsa"
 ### Example configuration
 
 For example, the configuration below is set in `cardhost/data-sources/default.js`.
-It would use a local git repository for local development,
-and point to a repository on GitHub once it is deployed to production.
-Remember, keep your private keys safe and do not commit them!
+The Hub has a `default-data-source` relationship that points to `@cardstack/git`.
```

Comment:
Perhaps instead:

> This config creates a `@cardstack/git` data source named `default` and configures it as the Hub's `default-data-source`.

Reply:
Changed to:


> For example, the configuration below is set in `cardhost/data-sources/default.js`.
> This config creates a `@cardstack/git` data source named `default` and configures it 
> as the Hub's `default-data-source`.
> Now, if a Card does not have its own data source defined, it will use git.